doc/README.md

```diff
@@ -404,6 +406,67 @@ click_info {
 
 ***
 
+
+## <a name="py3"></a>Py3 module helper
+
+Py3 is a special helper object that gets injected into
+py3status modules, providing extra functionality.  A module can access it via
+`self.py3` in its `Py3status` class
+
+#### Constants
+
+__CACHE_FOREVER__
+
+If this is returned as the value for `cached_until` then the module will not be
+updated.  This is useful for static modules and ones updating asynchronously.
+
+#### Methods
+
+__update(module_name=None)__
+
+Update a module.  If `module_name` is supplied the module of that
+name is updated.  Otherwise the module calling is updated.
+
+__get_output(module_name)__
+
+Return the output of the named module.  This will be a list.
+
+__trigger_event(module_name, event)__
+
+Trigger the event on named module
+
+__notify_user(msg, level='info')__
+
+Send notification to user.
+`level` can be `info`, `error` or `warning`
+
+__time_in(seconds=0)__
+
+Returns time seconds in the future.  Helpfull for creating cached_until
```

Comment:
→ Returns the time a given number of seconds into the future.  Helpful for creating the `cached_until` value for the module output.